diff --git a/lib/index.js b/lib/index.js
index 3de96ebf157df0abce33c358adc025ea90e99dda..10077e603ced66c41a1bceb235af45704d4bcacc 100644
--- a/lib/index.js
+++ b/lib/index.js
@@ -53,7 +53,7 @@ var SlashProvider = class {
 				(this.#root ?? view.dom.parentElement ?? document.body).appendChild(this.element);
 				this.#initialized = true;
 			}
-			if (composing || isSame) return;
+			if (isSame) return;
 			if (!this.#shouldShow(view, prevState)) {
 				this.hide();
 				return;
